src/components/Problems/Discussion.tsx

```diff
@@ -188,7 +188,11 @@ export const Discussion: FC<DiscussionProps> = ({problemId, invalidateSeriesQuer
                 onChange={handleCommentChange}
               />
               <Stack alignSelf="end">
-                <Button variant="button2" onClick={() => addComment()}>
+                <Button
+                  variant="button2"
+                  onClick={() => addComment()}
+                  disabled={isPending || commentText === '' || commentText == null}
```

Comment:
ja by som dal takto 😄 
```suggestion
                  disabled={isPending || !commentText}
```